examples/tina-self-hosted-demo/tina/config.tsx

```diff
@@ -1,35 +1,24 @@
 import { defineStaticConfig } from 'tinacms'
 
-const TINA_TOKEN_KEY = 'tina_token_key'
-
 import { contentBlockSchema } from '../components/blocks/content'
 import { featureBlockSchema } from '../components/blocks/features'
 import { heroBlockSchema } from '../components/blocks/hero'
 import { testimonialBlockSchema } from '../components/blocks/testimonial'
 import { ColorPickerInput } from '../components/fields/color'
 import { iconSchema } from '../components/util/icon'
+import { UsernamePasswordNextAuthProvider } from 'tinacms-next-auth/dist/tinacms'
+
+const isLocal = process.env.TINA_PUBLIC_IS_LOCAL === 'true'
 
 const config = defineStaticConfig({
   contentApiUrlOverride: '/api/gql',
-  admin: {
-    // auth: {
-    //   // If you wanted to use custom auth
-    //   getToken: async () => {
-    //     return { id_token: 'some-token' }
-    //   },
-    //   logout: async () => {
-    //     localStorage.removeItem(TINA_TOKEN_KEY)
-    //   },
-    //   authenticate: async () => {
-    //     localStorage.setItem(TINA_TOKEN_KEY, 'some-token')
-    //     return true
-    //   },
-    //   getUser: async () => {
-    //     return localStorage.getItem(TINA_TOKEN_KEY)
-    //   },
-    // },
-  },
   clientId: process.env.NEXT_PUBLIC_TINA_CLIENT_ID!,
+  authProvider:
+    !isLocal &&
+    new UsernamePasswordNextAuthProvider({
+      name: 'Credentials',
```

Comment:
What is `name: 'Credentials'`used for and could "Credentials" be a default value?